.gitignore

```diff
@@ -12,3 +12,4 @@
 /captures
 .externalNativeBuild
 .cxx
+/.idea/
```

Comment:
so because the .idea folder was already created, you'll also have to delete it and commit that deletion so it is no longer in the repo